easydock/database.py

```diff
@@ -136,23 +139,38 @@ def restore_setup_from_db(db_fname):
 
     return d, tmpfiles
 
+def get_isomers(mol, max_isomers):
+    opts = StereoEnumerationOptions(tryEmbedding=True, maxIsomers=max_isomers, rand=0xf00d)
+    # this is a workaround for rdkit issue - if a double bond has STEREOANY it will cause errors at
+    # stereoisomer enumeration, we replace STEREOANY with STEREONONE in these cases
+    try:
+        isomers = tuple(EnumerateStereoisomers(mol, options=opts))
+    except RuntimeError:
+        for bond in mol[1].GetBonds():
+            if bond.GetStereo() == Chem.BondStereo.STEREOANY:
+                bond.SetStereo(Chem.rdChem.BondStereo.STEREONONE)
+        isomers = tuple(EnumerateStereoisomers(mol,options=opts))
+    return isomers
 
-def init_db(db_fname, input_fname, prefix=None):
+def init_db(db_fname, input_fname, max_isomers, prefix=None):
```

Comment:
add please default value `max_isomers=1`
This will make the code more compatible with previous versions, because previously by default we generated one random stereoisomer